install.sh

```diff
@@ -33,7 +33,12 @@ yarn_get_tarball() {
     yarn_verify_integrity $tarball_tmp
 
     printf "$cyan> Extracting to ~/.yarn...$reset\n"
-    temp=$(mktemp -d) && tar zxf $tarball_tmp -C "$temp" && mkdir .yarn && mv "$temp"/*/* .yarn && rm -rf "$temp"
+    # All this dance is because `tar --strip=1` does not work everywhere
```

Comment:
I'd specifically mention Alpine - when I encounter this kind of comment I often wonder what environment is causing issues, so that I can check if that's still the case

Reply:
Good thinking but this may not be limited to Alpine. It is mostly all old versions of BusyBox that comes with their own `tar`.

Reply:
>  It is mostly all old versions of BusyBox 

Hmm, I wonder if I could install Yarn on a router. A large number of routes run BusyBox. It's good for environments with very little RAM 😛